e2e/pod.go

```diff
@@ -111,7 +133,19 @@ func getCommandInPodOpts(f *framework.Framework, c, ns string, opt *metav1.ListO
 }
 
 func execCommandInPod(f *framework.Framework, c, ns string, opt *metav1.ListOptions) (string, string, error) {
-	podPot, err := getCommandInPodOpts(f, c, ns, opt)
+	podPot, err := getCommandInPodOpts(f, c, ns, "", opt)
+	if err != nil {
+		return "", "", err
+	}
+	stdOut, stdErr, err := f.ExecWithOptions(podPot)
+	if stdErr != "" {
+		e2elog.Logf("stdErr occurred: %v", stdErr)
+	}
+	return stdOut, stdErr, err
+}
+
+func execCommandInContainer(f *framework.Framework, c, ns string, cn string, opt *metav1.ListOptions) (string, string, error) {
+	podPot, err := getCommandInPodOpts(f, c, ns, cn, opt)
```

Comment:
eventhough its not introduced here, Is above (`podPot`) a weird name ? may be we have to rename it as `podOpt` ?

Reply:
addressed this one too. Thanks!

Reply:
Cool. thanks !